src/ActionHandler/NotificationUpdateHandler.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace CommonGateway\VrijBRPToZGWBundle\ActionHandler;
+
+use CommonGateway\VrijBRPToZGWBundle\Service\NewSynchronizationService;
+use CommonGateway\VrijBRPToZGWBundle\Service\VrijBrpService;
+use CommonGateway\CoreBundle\ActionHandler\ActionHandlerInterface;
+
+class NotificationUpdateHandler implements ActionHandlerInterface
+{
+
+
+    public function __construct(
+        private readonly VrijBrpService $vrijBrpService,
+    ) {
+
+    }//end __construct()
+
+
+    /**
+     *  This function returns the requered configuration as a [json-schema](https://json-schema.org/) array.
+     *
+     * @throws array a [json-schema](https://json-schema.org/) that this  action should comply to
+     */
+    public function getConfiguration(): array
+    {
+        return [
+            '$id'         => 'https://commongateway.nl/ActionHandler/SynchronizationCollectionHandler.ActionHandler.json',
+            '$schema'     => 'https://docs.commongateway.nl/schemas/ActionHandler.schema.json',
+            'title'       => 'SynchronizationCollectionHandler',
+            'description' => '',
+            'required'    => [],
+            'properties'  => [],
```

Comment:
Missing properties in Handler here

Reply:
This action does not take configuration as of yet